spec/models/king_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  describe '#create' do
+    it 'creates a king in the database' do
+      king = FactoryGirl.create(:king)
+      expect(king.type == 'King').to eq(true)
+    end
+  end
+
+  describe '#valid_move?' do
+    it 'returns true if the move is diagonal up-right one square' do
+      king = FactoryGirl.create(:king, x_position: 4, y_position: 4)

```

Comment:
:-) Ha, I was just working on resolving the conflicts too, then got a message saying "This page is now out of date. Start over."

But since creating the PR I have edited (locally) the king factory in `pieces.rb` (to place the king at 4,4) and the **king =** line in each test in `king_spec.rb`. Don't I have to commit and push those changes to the remote **add_king_logic** branch before it's merged into master?